test/parzival/gamification_test.exs

```diff
@@ -6,33 +6,24 @@ defmodule Parzival.GamificationTest do
   describe "curriculums" do
     alias Parzival.Gamification.Curriculum
 
-    import Parzival.GamificationFixtures
-
-    @invalid_attrs %{summary: nil}
+    @invalid_attrs %{summary: 123}
 
     test "list_curriculums/0 returns all curriculums" do
-      curriculum = curriculum_fixture()
+      curriculum = Parzival.AccountsFixtures.user_fixture().curriculum
```

Comment:
Why?

Reply:
Because the previous solution was returning :
```
errors: [user_id: {"has already been taken", [validation: :unsafe_unique, fields: [:user_id]]}]
```